docs/src/imex/IMEXMultistep.md

```diff
@@ -0,0 +1,10 @@
+# IMEX Multistep Methods
+
+This article is a stub and likely needs a specialized example.
```

Comment:
@ChrisRackauckas

Reply:
The example in the tests was pretty meh, not really having a stiff and non-stiff part, but will have to do for now.